apis/core.oam.dev/common/types.go

```diff
@@ -112,6 +112,22 @@ type WorkloadGVK struct {
 	Kind       string `json:"kind"`
 }
 
+// DisruptiveProperty describes whether when a trait is applied to an Application,
```

Comment:
// DisruptiveProperty describes whether a trait will cause the pod to restart due to changes in some fields 
// when it is applied to an Application. 

Changing `when` order looks more clear.

Reply:
good idea